ev: announcement blog post (#422)
mixxxdj/website: content/news/XXXX-XX-XX-introducing-mixxx-ev.md

title: Introducing Mixxx e.V. - A Nonprofit Organization for the Community
authors: Mixxx Development Team
status: draft
tags: announcement, community, mixxx e.v.
comments: yes

> We are excited to announce the formation of **Mixxx e.V.** (eingetragener Verein), a nonprofit organization established to support and protect the Mixxx project and community. This marks an important milestone in Mixxx's journey as we continue our mission to make DJ software accessible to everyone.

Reviewer: ```suggestion
We are excited to announce the formation of **Mixxx e.V.**, a registered nonprofit association under German law, established to support and protect the Mixxx project and community. This marks an important milestone in Mixxx's journey as we continue our mission to make DJ software accessible to everyone.
```

Reviewer 2: The "(eingetragener Verein)" part is helpful to understand what the e.V. is for. I would keep it.

Author: applied manually

## What is Mixxx e.V.?

> Mixxx e.V. is a registered nonprofit association in Germany that serves as the legal body behind Mixxx DJ Software. The organization was founded in 2025 by Core development team members who recognized the need for a formal legal structure to better serve our growing community.

Reviewer: ```suggestion
Mixxx e.V. is a registered association (eingetragener Verein) that is recognized as nonprofit (gemeinnützig) by the German authorities. Mixxx e.V. serves as the legal body behind Mixxx DJ Software. The organization was founded in 2025 by Core development team members who recognized the need for a formal legal structure to better serve our growing community.
```

Author: applied manually

Our officially stated purpose is to promote art, culture, education, and occupational training by:

- Providing free and open source DJ software to artists regardless of their financial means
- Educating software developers in digital sound processing and online collaboration
- Supporting the creative expression and technical growth of our community

## Why Create a Nonprofit Organization?

Having a registered nonprofit entity brings several important benefits to the Mixxx project:

- **Legal Protection**: The e.V. shields individual community members and contributors from legal liability

> - **Financial Transparency**: We can now collect donations directly and manage funds with proper accountability

Reviewer: ```suggestion
- **Financial Transparency**: We can now collect donations directly and manage funds with proper accountability
-**Tax-deductible donations**: Donations to Mixxx e.V. are tax-deductible in Germany and many other European countries
```

The e.V. supports the development of Mixxx without exercising control over technical decisions or implementation details. Our community-driven, open-source development model remains unchanged.

## Who's Leading Mixxx e.V.?

After extensive discussion and collaboration on the Articles of Association, we held a founding meeting where all core members were invited to participate. The first Board was elected for a 3-year term expiring in 2028:

- **Owen Williams**: Chairperson
- **Jörg Wartenberg**: Second Chairperson
- **Daniel Schürmann**: Treasurer

## Donating to Mixxx

With Mixxx e.V. established, we now have more options for accepting donations from our community.
We continue to maintain our [US-based OpenCollective fundraising page](https://opencollective.com/mixxx) for donations in US dollars.
Additionally, Mixxx e.V. now has its own distinct [EU-based OpenCollective fundraising page](https://opencollective.com/mixxxev) that accepts donations in Euros.

> Donations to the US fundraiser will continue to directly fund development and promotion of Mixxx DJ Software, including purchasing hardware, funding appearances at conferences, and creating merchandise. Donations to the e.V. will also support Mixxx DJ Software in the same ways, and may additionally go to support any other efforts Mixxx e.V. pursues that promote its core mission.

Reviewer: ```suggestion
Donations to the US fundraiser will continue to directly fund development and promotion of Mixxx DJ Software, including purchasing hardware, funding appearances at conferences, and creating merchandise.
Donations to the e.V. will also support Mixxx DJ Software, but more restricted by the nonprofit purposes of Mixxx e.V.
```

Reviewer 2: Both are restricted kind of.

## Who's Leading Mixxx e.V.?

After extensive discussion and collaboration on the Articles of Association, we held a founding meeting where all core members were invited to participate. The first Board was elected for a 3-year term expiring in 2028:

- **Owen Williams**: Chairperson
- **Jörg Wartenberg**: Second Chairperson
- **Daniel Schürmann**: Treasurer

## Donating to Mixxx

With Mixxx e.V. established, we now have more options for accepting donations from our community.

We continue to maintain our [US-based OpenCollective fundraising page](https://opencollective.com/mixxx) for donations in US dollars.
Donations to this fundraiser will continue to directly fund development and promotion of Mixxx DJ Software, including purchasing hardware, funding appearances at conferences, and creating merchandise.

Additionally, Mixxx e.V. now has its own separate [EU-based OpenCollective fundraising page](https://opencollective.com/mixxxev) and a [PayPal account (url needed!)](https://paypal.com/?????). Donations to the e.V. will support Mixxx DJ Software as above, and may also go to support any other efforts Mixxx e.V. is interested in pursuing that promote its core mission.

> Anyone is welcome to donate to either organization depending on their preferred currency and payment method. All donations go toward supporting the development of Mixxx, maintaining our infrastructure, and funding community initiatives like internships and educational programs. Donating to OpenCollective with a direct bank transfer can help avoid credit card processing fees.

Reviewer 3: just a bit of consolidation. Formalities first, then the purpose/benefit
```suggestion
Additionally, Mixxx e.V. now has its own separate [EU-based OpenCollective fundraising page](https://opencollective.com/mixxxev) and a [PayPal account (url needed!)](https://paypal.com/?????). Anyone is welcome to donate to either organization depending on their preferred currency and payment method. Donating to OpenCollective with a direct bank transfer can help avoid credit card processing fees.

Donations to the e.V. will support Mixxx DJ Software as above, and may also go to support any other efforts Mixxx e.V. is interested in pursuing that promote its core mission. All donations go toward supporting the development of Mixxx, maintaining our infrastructure, and funding community initiatives like internships and educational programs.
```

Reviewer 3: Else LGTM, well written. Thanks!

Author: I did it that way because the first paragraph is about the e.v., so I didn't want to flop back and forth between what applies to both orgs vs what only applies to the ev.  I'll do another edit and see what you think.

Author: Punted all donations information to the donations page

## Donating to Mixxx

With Mixxx e.V. established, we now have more options for accepting donations from our community.
We continue to maintain our [US-based OpenCollective fundraising page](https://opencollective.com/mixxx) for donations in US dollars.
Additionally, Mixxx e.V. now has its own distinct [EU-based OpenCollective fundraising page](https://opencollective.com/mixxxev) that accepts donations in Euros.

Donations to the US fundraiser will continue to directly fund development and promotion of Mixxx DJ Software, including purchasing hardware, funding appearances at conferences, and creating merchandise. Donations to the e.V. will also support Mixxx DJ Software in the same ways, and may additionally go to support any other efforts Mixxx e.V. pursues that promote its core mission.

> Anyone is welcome to donate to either organization depending on their preferred currency and payment method. All donations go toward supporting the development of Mixxx, maintaining our infrastructure, and funding community initiatives like internships and educational programs. Donating with a direct bank transfer can help avoid credit card processing fees and allow Mixxx to retain more of your donation.

Reviewer 3: "All donations go toward supporting the development of Mixxx, maintaining our infrastructure, and funding community initiatives like internships and educational programs."

With this last it's confusing me tbh, because now we have
* "Donations to the US fundraiser will continue ...
* "Donations to the e.V. will also ...
* and the third one "All donations go toward supporting ...

Can you restructure/consolidate/remove redundancy so that we have the conlusion last:
"Anyone is welcome to donate to either organization depending on their preferred currency and payment method."

And move this up, right after the US/eV sentences
"Donating with a direct bank transfer can help avoid credit card processing fees and allow Mixxx to retain more of your donation."

Reviewer 2: The thing is actually a bit different. We have two organizations now, the Mixxx e.V. (German non profit) and the Opens Source collective (US non profit) 
Both have their own non profit purpose that restricts the usage of the donated money. 
Mixxx e.V. -> Promote art, culture and education by publishing the software Mixxx
Open Source Collective -> Support the Open Source Ecosystem of the software Mixxx.  

> Our mission is to support a sustainable and healthy open source ecosystem by providing an umbrella structure for our hosted projects and handling the behind-the-scenes tasks such as issuing invoices, managing a bank account, contract review and signature, and more. 

As e.V. we can forward bills related to the "Open Source Ecosystem", to Open Source Collective. They refund them after their approval. We have no access to the money directly. 

Not sure how much details we want here in the news article. We just do not give the impression the Open Source Collective money is ours to avoid tax issues.

Reviewer 2: This one is open. Let's reduce it here and forward to our new donations page.
